### DISCUSSION TITLE
New update offsets.json code for build 13965 [working perfect for me]

### DISCUSSION THREAD

**Discussion by tkqu** [IMXNOOBX/cs2-external-esp#56] [Show and tell]
{
    "build_number": 13965,
    "dwBuildNumber": 4748564,
    "dwEntityList": 24745408,
    "dwLocalPlayer": 25067864,
    "dwPlantedC4": 25739232,
    "dwViewMatrix": 25720624,
    "m_ArmorValue": 5360,
    "m_bIsDefusing": 5008,
    "m_flC4Blow": 3760,
    "m_flNextBeep": 3756,
    "m_flTimerLength": 3768,
    "m_hPlayerPawn": 1980,
    "m_iAccount": 64,
    "m_iHealth": 812,
    "m_iTeamNum": 959,
    "m_pClippingWeapon": 4752,
    "m_pGameSceneNode": 784,
    "m_pInGameMoneyServices": 1744,
    "m_sSanitizedPlayerName": 1824,
    "m_szName": 3096,
    "m_vecAbsOrigin": 200,
    "m_vecOrigin": 4628
}

**Comment by ifBars** [comment]
> { "build_number": 13965, "dwBuildNumber": 4748564, "dwEntityList": 24745408, "dwLocalPlayer": 25067864, "dwPlantedC4": 25739232, "dwViewMatrix": 25720624, "m_ArmorValue": 5360, "m_bIsDefusing": 5008, "m_flC4Blow": 3760, "m_flNextBeep": 3756, "m_flTimerLength": 3768, "m_hPlayerPawn": 1980, "m_iAccount": 64, "m_iHealth": 812, "m_iTeamNum": 959, "m_pClippingWeapon": 4752, "m_pGameSceneNode": 784, "m_pInGameMoneyServices": 1744, "m_sSanitizedPlayerName": 1824, "m_szName": 3096, "m_vecAbsOrigin": 200, "m_vecOrigin": 4628 }

The proper way to do this, would be to create a fork, update them in offsets.json, then create a pull request to merge. That way all we need to do is press merge, and it will update them, along with it will actually put you as a part of the contributors for updating the offsets. For future reference, feel free to contact me on discord, found on my profile here, and I would be happy to walk you through creating a pull request to update offsets. Will test them soon and create a pull request if no one else does in time.
